spec/rubocop/cop/rspec/sort_metadata_spec.rb

```diff
@@ -100,27 +148,27 @@
      'and the hash values are complex objects' do
     expect_offense(<<~RUBY)
       it 'Something', variable, 'B', :a, key => {}, foo: ->(x) { bar(x) }, Identifier.sample => true, baz: Snafu.new do
```

Comment:
This works for "it". Confusing, but I recall that we've explored this diverged behaviour.

Reply:
yes, that's a bit unsatisfying.